netmanager/src/views/pages/Download/Download.js

```diff
@@ -186,7 +258,98 @@ const Download = (props) => {
       })
       .catch((err) => console.log(err && err.response && err.response.data));
     setLoading(false);
+    setStartDate(null);
+    setEndDate(null);
+    setFileType(null);
+    setSelectedSites([]);
+    setPollutants([]);
   };
+
+  const handleUrbanBetterFormSubmit = async (e) => {
+    e.preventDefault();
+
+    setLoading(true);
+
+    let data = {
```

Comment:
We could change this to a `const` just in case we do not expect it to change within the current function scope